README.md

```diff
@@ -112,7 +121,11 @@ See the readme file in the ```./portable``` directory for more information.
 
 - The ```./include``` directory contains the real time kernel header files.
 
+- The ```./sample_configuration``` directory contains sample a `FreeRTOSConfig.h` to help jumpstart a new project.
+See the `FreeRTOSConfig.h` file for instructions.
```

Comment:
We can make this into a link by giving a relative path like so:

[MyFile.h](portable/myDirectory/MyFile.h)

This is something which GitHub understands AFAIK.